src/constants.ts

```diff
@@ -0,0 +1,4 @@
+type OperatorSymbols = ["/", "X", "-", "+"];
+
+export const numberPads = ["9", "8", "7", "6", "5", "4", "3", "2", "1", "0"];
+export const operators: OperatorSymbols = ["/", "X", "-", "+"];
```

Comment:
이제보니.. 중복 선언이 아니라 타입 자체를 배열로 선언했군요.
어차피 이거 다른데서 쓰이는게 아니라서

```suggestion
export const numberPads = ["9", "8", "7", "6", "5", "4", "3", "2", "1", "0"] as const;
export const operators = ["/", "X", "-", "+"] as const;
```
이렇게 const로 선언해주면 된답니다.